.github/workflows/sandbox.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('go.sum') }}
       - name: Build Flytectl binary
         run: make compile
+      - name: Setup env
+        run: |
+          mkdir -p ~/.flyte/k3s
+          touch ~/.flyte/k3s/k3s.yaml
+          chmod 777 ~/.flyte/k3s/k3s.yaml
```

Comment:
This requires only readwrite and not execute

Reply:
yes

Reply:
Can we Reduce the permissions then